src/link-handler.js

```diff
@@ -92,6 +92,10 @@ export class DefaultLinkHandler extends LinkHandler {
       return info;
     }
 
+    if (target.hasAttribute('download') || target.hasAttribute('router-ignore')) {
```

Comment:
I'll need to think about a "router-ignore" attribute.

Reply:
Ok, my other possible solution was to just ignore any links with a target attribute, but I thought you might not like that.